### PR TITLE
Handle labels on highlights

### PR DESCRIPTION
**Pull request by jacksonh**
Pull in labels from highlights: now that highlights have labels
   we can pull them in as tags, I used the `#[[ name ]]` syntax
   as this allows labels with spaces in their name.

### CODE REVIEW

**Comment by jacksonh** [comment]
Need to fix some escaping issues.

**Comment by jacksonh** [comment]
OK, this grew a bit. I included a fix for what I think is the issue causing Paul's sync to fail, discussed here: https://discord.com/channels/844965259462311966/943967463299764314/1051843289575329822

**Comment by jacksonh** [comment]
Going to try and rebase and fix this one

**Comment by jacksonh** [comment]
This PR is simplified now and just does the labels import, so I have adjusted the title and description

File: src/index.ts [hunk 1]
```diff
@@ -463,7 +470,7 @@ const main = async (baseInfo: LSPluginBaseInfo) => {
       title: 'Enter the template to use for new highlights',
       description:
         'Enter the template to use for new highlights. Required variables are: {{{text}}}, {{{highlightUrl}}}. Optional variables are {{{dateHighlighted}}}',
-      default: `> {{{text}}} [⤴️]({{{highlightUrl}}})`,
+      default: `> {{{text}}} [⤴️]({{{highlightUrl}}}) {{#labels}}#[[{{{name}}}]]{{/labels}}`,
```
Comment (sywhb) [inline src/index.ts:473]: I guess we could add a space in between the labels otherwise it could be a little squeezy if there are many

**Review by sywhb — APPROVED** [review]
LGTM

File: src/index.ts (hunk 1 above)
Comment (sywhb) [reply src/index.ts:473]: Screenshot 2022-12-30 at 4 10 58 PM

Comment (jacksonh) [reply src/index.ts:473]: Good call on the space, my theme kind of had some padding so I didn't notice.

**Pull request merged**